pkg/network/self_sign_configuration.go

```diff
@@ -0,0 +1,141 @@
+package network
+
+import (
+	"reflect"
+	"time"
+
+	"github.com/pkg/errors"
+
+	opv1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+)
+
+const (
+	caRotateIntervalDefault   = 7 * 24 * time.Hour // 7 days
+	caOverlapIntervalDefault  = 24 * time.Hour     // 1 day
+	certRotateIntervalDefault = 24 * time.Hour     // 1 day
```

Comment:
why is this the default? are we aligned to kubevirt intervals?

Reply:
Yep we are aligned with them